canonical-web-and-design/react-components
# Add Checkbox and Radio components

## Done

- Added Checkbox and Radio components, as well as a `CheckableInput` wrapper component

## QA

- Visit [Checkbox docs](https://react-components-441.demos.haus/?path=/story/checkboxinput--default-story)
- See that each state looks good
- Visit [Radio docs](https://react-components-441.demos.haus/?path=/docs/radioinput--default-story)
- See that each state looks good
- See that all tests pass

### Storybook

To see rendered examples of all react-components, run:

```shell
yarn start
```

### QA in your project

from `react-components` run:

```shell
yarn build
npm pack
```

Install the resulting tarball in your project with:

```shell
yarn add <path-to-tarball>
```

## Fixes

Fixes: #259 & #418  .
